packages/vx-responsive/src/enhancers/withScreenSize.tsx

```diff
@@ -37,6 +37,7 @@ export default function withScreenSize<Props extends WithScreenSizeProps = {}>(
 
     componentWillUnmount() {
       window.removeEventListener('resize', this.handleResize, false);
+      this.resize.cancel();
```

Comment:
ditto re updating this to `this.handleResize.cancel()` since the unwrapped method wouldn't have the `.cancel()` method